frame/wormhole/backing/s2s/src/lib.rs

```diff
@@ -91,30 +97,44 @@ pub mod pallet {
 		type OutboundPayload: Parameter + Size;
 		type CallEncoder: EncodeCall<Self::AccountId, Self::OutboundPayload>;
 
-		type FeeAccount: Get<Option<Self::AccountId>>;
-		type MessageSender: RelayMessageSender;
-		type MessageSendPalletIndex: Get<u32>;
-		type MessageLaneId: Get<[u8; 4]>;
+		type MessageNoncer: LatestMessageNoncer;
+		type MessageLaneId: Get<LaneId>;
+
+		type OutboundMessageFee: Default
+			+ From<u64>
+			+ PartialOrd
+			+ Parameter
+			+ Saturating
+			+ Zero
+			+ Copy
+			+ From<RingBalance<Self>>;
+
+		type MessagesBridge: MessagesBridge<
+			Self::AccountId,
+			Self::OutboundMessageFee,
```

Comment:
Can we use `type RingCurrency` to replace `Self::OutboundMessageFee`. I think those two items are equal.

Reply:
maybe we can use RingBalance